free-courses-en.md

```diff
@@ -97,6 +98,9 @@
 * [Bento Shell Track](https://bento.io/topic/shell) (Bento)
 * [Shell Scripting Tutorial](https://www.youtube.com/playlist?list=PL7B7FA4E693D8E790)
 
+### BlockChain
+* [Bitcoin and Cryptocurrency Technologies](https://www.lopp.net/pdf/princeton_bitcoin_book.pdf) - Bitcoin and Cryptocurrency Technologies
+* [Official BlockChain guide](https://bitcoin.org/en/blockchain-guide)
```

Comment:
good resource.

Reply:
The url was changed. New link is [here](https://developer.bitcoin.org/devguide/index.html)

Reply:
Maybe give this PR a fresh start?

Reply:
I think we shall do that. I will close this PR, and work on a new one. Also I am a bit tight on time these days, I will work on enhancements not immediately. I hope you understand 😄  .